src/python/ensembl/io/genomio/fasta/process.py

```diff
@@ -96,6 +99,13 @@ def main() -> None:
     parser.add_argument_src_path("--genbank_infile", help="Input GenBank GBFF file")
     parser.add_argument_dst_path("--fasta_outfile", required=True, help="Output FASTA file")
     parser.add_argument("--peptide_mode", action="store_true", help="Process proteins instead of DNA")
+    parser.add_log_arguments()
```

Comment:
```suggestion
    parser.add_log_arguments(add_log_file=True)
```
You need to set this flag for the argument parser to offer the `log_file` and `log_file_level` options.